neusomatic/python/call.py

```diff
@@ -287,17 +290,24 @@ def pred_vcf_records(ref_file, final_preds, true_path, chroms, vartype_classes,
     pool = multiprocessing.Pool(num_threads)
     try:
         all_vcf_records = pool.map_async(pred_vcf_records_path, map_args).get()
-        all_vcf_records = filter(lambda x: x, all_vcf_records)
         pool.close()
     except Exception as inst:
         logger.error(inst)
         pool.close()
         traceback.print_exc()
         raise Exception
+
+    for o in all_vcf_records:
+        if o is None:
+            raise Exception("pred_vcf_records_path failed!")
+
+    all_vcf_records = filter(None, all_vcf_records)
```

Comment:
This filter would never remove anything since exception would be thrown for None entries. So, seems superfluous.

Reply:
The exceptions is when we have None. But, the filter is to remove [ ].